lib/access_import/access_import.rb

```diff
@@ -75,6 +76,25 @@ def make_assignments
       assignment.creator_id = @import_user.id
       assignment.save!
     end
+    puts "Imported #{Assignment.count - records_before} new #{Assignment.class.name} "\
+      'from MS Access Database.'
```

Comment:
More readable:

```ruby
puts "Imported %d new %s from MS Access Database." %
  [Assignment.count - records_before, Assignment.name]
```

(you probably meant `Assignment.name`, `Assignment.class.name` is always "Class" ;-))